build.py

```diff
@@ -857,6 +862,13 @@ def container_build(images, backends, repoagents):
         action="store_true",
         required=False,
         help='Do not use Docker --pull argument when building container.')
+    parser.add_argument(
+        '--target-platform',
+        required=False,
+        default=None,
+        help=
+        'Target Platform, can be "ubuntu", "windows" or "jetpack". If not specified, auto-detect the same.'
```

Comment:
"Target for build, can be ... If not specified, build targets the current platform"